source/common/grpc/google_async_client_impl.cc

```diff
@@ -84,7 +84,7 @@ GoogleAsyncClientImpl::GoogleAsyncClientImpl(Event::Dispatcher& dispatcher,
   stub_ = stub_factory.createStub(channel);
   // Initialize client stats.
   stats_.streams_total_ = &scope_->counter("streams_total");
-  for (uint32_t i = 0; i <= Status::GrpcStatus::MaximumValid; ++i) {
+  for (uint32_t i = 0; i <= Status::WellKnownGrpcStatus::UserDefinedGrpcStatus; ++i) {
```

Comment:
I will probably keep this until MaximumValid, until we see some feature request for envoy grpc client. I don't think you increment this counter either?